### DISCUSSION TITLE
How do I disable the features in Windows Explorer?

### DISCUSSION THREAD

**Discussion by will64gamer** [Open-Shell/Open-Shell-Menu#579] [General]
By that I mean the top (where I gather the OpenShell toolbar for Windows Explorer should be but isn't showing up) and bottom bars which appear in white on this screeshot:
![image_2020-12-17_19-07-14](https://user-images.githubusercontent.com/58888901/102554509-60054a00-409b-11eb-8c4a-bcb5c77d06f2.png)

**Comment by bonzibudd** [comment]
Just run the installer again, click "Change" and de-select "Classic Explorer."

**Reply by will64gamer** [reply]
Thank you, that did the trick for the bottom bar, the line at the top seems not to be Open-Shell related.
Edit: Yup, turns out the white line at the top was a bug with windows dark theme (if anyone needs to solve it check [here](https://answers.microsoft.com/en-us/windows/forum/windows_10-start-win_theme/how-to-remove-white-line-on-explorer-dark-theme/2b8f78b9-23ca-4e78-a982-a9bb1584ee80/).)

**Comment by blackcrack** [comment]
@will64gamer, please if you want make the next post, can you take the other button, "post and close the issue"
thank's
